media-tv/mythtv/mythtv-30.0.ebuild

```diff
@@ -0,0 +1,408 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+# Specify a specific commit to download.  If we just pull the latest from the branch, the ebuild
+# digest will break.  Update this periodically.  When updating, keep old versions so that the
+# ebuild will work on all of them with simple renaming.
+BACKPORTS="5cde0578d84926171b20c8f7e95a101e9b0b9457" # August 8, 2019
+
+MY_P=${P%_p*}
+MY_PV=${PV%_p*}
+
+PATCHES=(
+	# Selected a fixes/... branch commmit to get rid of previous patches
+)
+
+inherit eutils flag-o-matic python-single-r1 qmake-utils user readme.gentoo-r1 systemd vcs-snapshot
+
+MYTHTV_BRANCH="fixes/${P%.*}"
+
+DESCRIPTION="Homebrew PVR project"
+HOMEPAGE="https://www.mythtv.org"
+SRC_URI="https://github.com/MythTV/mythtv/archive/${BACKPORTS}.tar.gz -> ${PF}.tar.gz"
```

Comment:
If you use this, you'll need to rename the ebuild having some _p appendix.
https://devmanual.gentoo.org/ebuild-writing/file-format/index.html